src/services/user.service.js

```diff
@@ -0,0 +1,14 @@
+import db from '../database/models';
+
+const UserService = {
+  getUserLineManager: async (id) => db.Users.findOne({ where: { id }, attributes: ['lineManager'] }),
+
+  getuserById: async (id) => db.Users.findOne({ where: { id } }),
```

Comment:
great job @urbanishimwe hower try to make the  function universal but adding a condition inside because not everyone has to search by id

Reply:
- I think we can accept the method. If we need to get users by other attributes, we can create another method. That would be more like a search functionality. @swaibat 
- Update the method name to be in `camelCase` @urbanishimwe

Reply:
alright

Reply:
I already made changes @swaibat  requested,

Reply:
because if someone wants to query a user by id with the status he will have to create a different function for it